src/Altinn.App.Core/Internal/Process/Elements/UserAction.cs

```diff
@@ -0,0 +1,31 @@
+using System.Runtime.Serialization;
+using System.Text.Json.Serialization;
+using Altinn.App.Core.Internal.Process.Elements.AltinnExtensionProperties;
+
+namespace Altinn.App.Core.Internal.Process.Elements
+{
+    /// <summary>
+    /// Defines an altinn action for a task
+    /// </summary>
+    public class UserAction
+    {
+        /// <summary>
+        /// Gets or sets the ID of the action
+        /// </summary>
+        [JsonPropertyName("id")]
+        public string Id { get; set; }
```

Comment:
I'm itching in my fingers to add this everywhere. Just curious when it will cause bugs.
```suggestion
        public required string Id { get; set; }
```